### DISCUSSION TITLE
Custom fonts not loaded

### DISCUSSION THREAD

**Discussion by ariefsn** [mjmlio/mjml#2588] [Q&A] [answered]
Hi.
I try to create some texts with google fonts.
Its working properly when convert to html.
But when its sent to email, somehow the font is not loaded.
Tried in Outlook and Gmail.
here the sample  [https://mjml.io/try-it-live/r8BIUNpmG](https://mjml.io/try-it-live/r8BIUNpmG)
and here the html version [https://jsfiddle.net/grtw95jm/](https://jsfiddle.net/grtw95jm/)

Thanks

**Comment by iRyusa** [comment, marked as answer]
Both doesn't support external font that's why.

**Reply by ariefsn** [reply]
oohh ok.
do you know email provider that support custom fonts?

**Reply by jpagano** [reply]
https://www.caniemail.com/features/css-at-font-face/

**Comment by iRyusa** [comment]
Apple mail does as far as I remember
